python/ccxt/async_support/base/ws/aiohttp_client.py

```diff
@@ -71,7 +73,7 @@ def create_connection(self, session):
         # otherwise aiohttp's websockets client won't trigger WSMsgType.PONG
         # call aenter here to simulate async with otherwise we get the error "await not called with future"
         # if connecting to a non-existent endpoint
-        return session.ws_connect(self.url, autoping=False, autoclose=False, headers=self.options.get('headers')).__aenter__()
+        return session.ws_connect(self.url, autoping=False, autoclose=False, headers=self.options.get('headers'), proxy = self.proxy).__aenter__()
```

Comment:
`proxy` can be set from outside explicitly, at i.e. 389th line of `async/exchange.py`